version_template.py

```diff
@@ -107,14 +102,34 @@ def buildNumber():
 
 
 def currentCommit():
+    import os
     from subprocess import check_output
-    return check_output('git log --pretty=oneline -n 1 -- $(pwd)', shell=True).split()[0]
+
+    return check_output(['git', 'log', '--pretty=oneline', '-n', '1', os.getcwd()]).split()[0]
 
 
 def dirty():
```

Comment:
A purist would commit the change to this method in a separate PR.

Reply:
I will do this in the future then.